addons/website_sale/views/templates.xml

```diff
@@ -213,7 +213,7 @@
             <t t-set="bg_color" t-value="td_product['ribbon']['bg_color'] or ''"/>
             <t t-set="text_color" t-value="td_product['ribbon']['text_color']"/>
             <t t-set="bg_class" t-value="td_product['ribbon']['html_class']"/>
-            <span t-attf-class="o_ribbon #{bg_class}" t-attf-style="#{text_color and ('color: %s; ' % text_color)}#{bg_color and 'background-color:' + bg_color}" t-raw="td_product['ribbon']['html'] or ''"/>
+            <span t-attf-class="o_ribbon #{bg_class}" t-attf-style="color: {{text_color or 'inherit'}}; background-color: {{bg_color or 'inherit'}}" t-esc="td_product['ribbon']['html'] or ''"/>
```

Comment:
it is not the same ! 
https://drive.google.com/file/d/1pXgTvjDjIEj2diYRAvjN31pMHJMZRn9n/view